### ISSUE TITLE
PPSSPP v1.9.4 still has black screen issue on Xiaomi Mi Box S.

### ISSUE THREAD

**Issue by backskin** [hrydgard/ppsspp#12860]
### What happens?
Downloaded PPSSPP from official Google Play Store (Even buy gold version), but got this 'cursed' issue with black screen every time I load any game. No software settings managed this problem, I tried literally every adjustment, every button. I've already read a thread about this issue, which is closed now. Please open it again.

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.
Mi Box S, regular one. Android 9, last update
PPSSPP version 1.9.4 from Google Play Market

**Comment by unknownbrackets** [comment]
Closing as a duplicate of #11922.

PPSSPP v1.9.4 will never work on Xiaomi Mi Box S.  We already released that version.  A future version, or git builds, will work.

We don't retroactively update old versions.  If we did, we'd still be on v0.3.0 because there'd be no reason to change the version number.  We'd just keep changing what v0.3.0 means.  Maybe then we'd invent something else, like a "change number" and say we were on "v0.3.0 changed 40 times".  In that case, we'd still never retroactively modify "v0.3.0 changed 39 times", right?  I think you see where I'm going.

If you're asking when the next stable release, including this fix, will be in the Google Play Store, that's currently slated for sometime between 2020-05-04 and 2030-05-08.  A date will be decided when we feel like it's ready for a release, which might be 1-2 weeks before that date.

-[Unknown]
